pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -51,7 +51,9 @@ type virtIOInterfaceManager struct {
 const (
 	// ReservedInterfaces represents the number of interfaces the domain
 	// should reserve for future hotplug additions.
-	ReservedInterfaces = 4
+	ReservedInterfaces            = 4
+	LibvirtInterfaceLinkStateUP   = "up"
+	LibvirtInterfaceLinkStateDown = "down"
```

Comment:
As the constants are only used in this file - they could be unexported.

Reply:
In "virtwrap, nichotplug: Hotplug interface in down state" it is still exported, but changed in "virtwrap, nichotplug: Live toggle of interface state"